docs/pscripts/pcreate.rst

```diff
@@ -6,8 +6,7 @@
 ``pcreate``
 -----------
 
-.. program-output:: pcreate --help
-   :prompt:
-   :shell:
+.. autoprogram:: pyramid.scripts.pcreate:PCreateCommand
+   :prog: pcreate.py
```

Comment:
Depending on the rendering output we probably want this to just say `pcreate` as that's the name of the program.